FFXIVClientStructs/FFXIV/Component/GUI/AtkTextureResourceManager.cs

```diff
@@ -13,6 +14,20 @@ public unsafe partial struct AtkTextureResourceManager {
     // Icon ranges are inclusive: id ∈ [a,b]
     [FieldOffset(0x30), FixedSizeArray] internal FixedSizeArray4<StdPair<uint, uint>> _localizedIconRange;
     [FieldOffset(0x50)] public uint LocalizedIconRangeCount;
+
+    /// <summary>
+    /// Unloads the texture if the reference count is 1, otherwise decrements the textures reference count.
+    /// </summary>
+    /// <param name="textureHandle">Pointer to a texture resource</param>
+    /// <returns>-1 if the textureHandle is null, 0 otherwise</returns>
+    [MemberFunction("E8 ?? ?? ?? ?? 32 C0 45 84 FF")]
+    public partial long UnloadTexture(TextureResourceHandle* textureHandle);
+
+    [MemberFunction("E8 ?? ?? ?? ?? 48 8B F0 48 85 C0 74 35 48 8B 48 08")]
+    public partial AtkTextureResource* LoadIconTexture(int iconId, IconSubFolder folder);
+    
+    [MemberFunction("E8 ?? ?? ?? ?? 4C 8B E8 48 85 C0 74 52")]
+    public partial AtkTextureResource* LoadTexture(CStringPointer path, IconSubFolder folder);
```

Comment:
- Missing the attribute to generate string overloads.
- Second argument is saved in `Version`, which is the texture scale. If not 1 (means it's 2, because scale 0 doesn't exist), it inserts `_hr1` to the path.

```suggestion
    [MemberFunction("E8 ?? ?? ?? ?? 4C 8B E8 48 85 C0 74 52"), GenerateStringOverloads]
    public partial AtkTextureResource* LoadTexture(CStringPointer path, int textureScale);
```